docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -1,38 +1,64 @@
+title: Test a .NET Standard class library with .NET Core in Visual Studio
 description: Create a unit test project for your .NET Core class library. Verify that your .NET Core class library works correctly with unit tests.
-author: BillWagner
-ms.author: wiwagn
-ms.date: 08/07/2017
+ms.date: 12/23/2019
 dev_langs:
   - "csharp"
   - "vb"
 ms.custom: "vs-dotnet, seodoc18"
 ---
-# Test a .NET Standard library with .NET Core in Visual Studio 2017
+# Test a .NET Standard library with .NET Core in Visual Studio
 
-In [Build a .NET Standard library with .NET Core in Visual Studio](library-with-visual-studio.md), you created a simple class library that adds an extension method to the <xref:System.String> class. Now, you'll create a unit test to make sure that it works as expected. You'll add your unit test project to the solution you created in the previous article.
+In [Build a .NET Standard library in Visual Studio](library-with-visual-studio.md), you created a simple class library that adds an extension method to the <xref:System.String> class. Now, you'll create a unit test to make sure that it works as expected. You'll add your unit test project to the solution you created in the previous article.
 
-## Creating a unit test project
+## Create a unit test project
 
 To create the unit test project, do the following:
 
-<!-- markdownlint-disable MD025 -->
+1. Open the `ClassLibraryProjects` solution you created in the [Build a .NET Standard library in Visual Studio](library-with-visual-studio.md) article.
 
-# [C#](#tab/csharp)
+1. Add a new unit test project named "StringLibraryTest" to the solution.
 
-1. In **Solution Explorer**, open the context menu for the **ClassLibraryProjects** solution node and select **Add** > **New Project**.
+   1. Right-click on the solution in **Solution Explorer** and select **Add** > **New project**.
 
-1. In the **Add New Project** dialog, select the **Visual C#** node. Then select the **.NET Core** node followed by the **MSTest Test Project (.NET Core)** project template. In the **Name** text box, enter "StringLibraryTest" as the name of the project. Select **OK** to create the unit test project.
+   1. On the **Add a new project** page, enter **mstest** in the search box. Choose **C#** or **Visual Basic** from the Language list, and then choose **All platforms** from the Platform list. Choose the **MsTest Test Project (.NET Core)** template, and then choose **Next**.
 
-   ![Add New Project dialog with unit test project displayed - C#](./media/testing-library-with-visual-studio/create-new-test-project.png)
+   1. On the **Configure your new project** page, enter **StringLibraryTest** in the **Project name** box. Then, choose **Create**.
 
    > [!NOTE]
-   > In addition to an MSTest Test project, you can also use Visual Studio to create an xUnit test project for .NET Core.
+   > In addition to an MSTest, you can also create xUnit and nUnit test projects for .NET Core in Visual Studio.
```

Comment:
This note suggests you might want to add multiple test projects using different frameworks. It would be more helpful if it gave some clues as to why you might want to do that.  That goes beyond the scope of this PR, though.

Reply:
Very good point and I couldn't find this info in any of our docs here for .NET or VS. Something for us to consider later.